src/locale/i18n.ts

```diff
@@ -28,7 +28,8 @@ import {messages as messagesId} from '#/locale/locales/id/messages'
 import {messages as messagesIt} from '#/locale/locales/it/messages'
 import {messages as messagesJa} from '#/locale/locales/ja/messages'
 import {messages as messagesKo} from '#/locale/locales/ko/messages'
-import {messages as messagesNl} from '#/locale/locales/nl/messages'
+import { messages as messagesNl } from '#/locale/locales/nl/messages'
+import {messages as messagesNp} from '#/locale/locales/id/messages'
```

Comment:
```suggestion
import {messages as messagesNl} from '#/locale/locales/nl/messages'
import {messages as messagesNp} from '#/locale/locales/np/messages'
```
A couple of extra spaces seem to have been added here by accident, and also the language codes seem to have gotten a bit mixed up :)